lib/Models/FunctionParameter.js

```diff
@@ -204,6 +210,11 @@ FunctionParameter.prototype.load = function() {
     return this._loadingPromise;
 };
 
+/**
+ * Represents value as string.
+ * @param {Object} value Value to format as string.
+ * @return {String} String representing value.
+ */
 FunctionParameter.prototype.formatValueAsString = function(value) {
     return defined(value) ? value.toString() : '-';
```

Comment:
This should use the parameter's own value if one isn't passed in, just like you did in the derived parameter types.